src/classes/RD2_RecurringDonations_TEST.cls

```diff
@@ -1021,6 +1067,7 @@ private with sharing class RD2_RecurringDonations_TEST {
             new String[]{ 
                 SObjectType.npe03__Recurring_Donation__c.fields.npe03__Installment_Period__c.getLabel() 
                 + ', ' + SObjectType.npe03__Recurring_Donation__c.fields.InstallmentFrequency__c.getLabel() 
+                + ', ' + SObjectType.npe03__Recurring_Donation__c.fields.Day_of_Month__c.getLabel() 
```

Comment:
@force2b , @andrewyu-salesforce : FYI - Pause RD2 unit test was updated to include the "Day of Month" field in the exception message since the field has been changed by the trigger which was  not the case when the test was originally written.